content/zh/docs/contributing/_index.md

```diff
@@ -0,0 +1,572 @@
+---
+title: 贡献
+description: 了解如何为OpenTelemetry文档做贡献.
+aliases: [/docs/contribution-guidelines]
```

Comment:
```suggestion
```

Reply:
yes,this should be removed. I have removed it in the latest commit with other changes together, thank you~